lib/yui_rest_client/widgets.rb

```diff
@@ -20,7 +20,6 @@ module Widgets
     require 'yui_rest_client/widgets/table'
     require 'yui_rest_client/widgets/textbox'
     require 'yui_rest_client/widgets/timefield'
-    require 'yui_rest_client/widgets/label'
```

Comment:
is intended? we still have spec for it, seems useful for reading text on the screen AFIR.

Reply:
I don't know why rubocop didn't complain before, but we had this import twice, so it's a fix.